### PR TITLE
gist: enhance list/update description

### PR DESCRIPTION
**Pull request by nicokosi**
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#commit-message).
- [x] The page follows the [content guidelines](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#guidelines).

### CODE REVIEW

File: pages/common/gist.md [hunk 1]
```diff
@@ -23,10 +23,10 @@
 
 `gist -l`
 
-- List all gists for the currently logged in user:
+- List all public gists for any user:
```
Comment (nicokosi) [inline pages/common/gist.md:26]: The command to list gists for currently logged user is just above : 

> - List your public and private gists:
>
 > `gist -l`

**Comment by bl-ue** [comment]
@nicokosi will you please expand all short options to long ones in the page where possible?

**Comment by nicokosi** [comment]
> @nicokosi will you please expand all short options to long ones in the page where possible?

Done in commit 18aafe02958dd32b42997d8e9c901978f8fce266.

**Review by bl-ue — APPROVED** [review]

**Review by glenacota — APPROVED** [review]
LGTM 👍

**Pull request merged**